main.rb

```diff
@@ -27,21 +39,19 @@ def initialize
   end
 
   def run
-    @retrieve_data.retrieve_all(@game_manager, @music_album_manager, @book_manager)
-    puts ''
-    puts 'WELCOME TO MY-CATALOG APP'
-    puts ''
+    @retrieve_data.retrieve_all(@game_manager, @music_album_manager)
+    selected_option = nil
```

Comment:
On Startup there is an Argument error, seems like you deleted the @book_manager argument

Reply:
Hey @sunga12, This is because your store data branch was not merged into the `develop `branch when I created this branch That's why it didn't merge your changes in it but now I have merged the `develop `branch and it should solve the problem 😊
thanks for mentioning 😀